index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/8361b8a285.js" crossorigin="anonymous"></script>
+    <script src="script.js" defer></script>
+    <title>Kano Global Summit</title>
+</head>
+<body>
+    <main class="main">
+        <section class="desktop-menu-bar">
+            <div class="top-bar">
+                <ul class="top-bar-elements">
+                <li><i class="fa-brands fa-facebook-f social-bar"></i></li>
+                <li><i class="fa-brands fa-twitter social-bar"></i></li>
+                <li><h2 class="bar-text">English</h2></li>
+                <li><h2 class="bar-text">My Page</h2></li>
+                <li><h2 class="bar-text">Layout</h2></li>
+                </ul>
+            </div>
+            <div class="manu-bar">
+                <ul class="menu-bar-items">
+                    <li class="summit-bar-logo"><a href="index.html"><img class="main-kano-logo" src="./img/global-summit-logo.png" alt="" ></a></li>
+                    <li><a class="link-to-about" href="about.html"><h2 class="menu-item">About</h2></li></a>
+                    <li><h2 class="menu-item">Program</h2></li>
+                    <li><h2 class="menu-item">Join</h2></li>
+                    <li><h2 class="menu-item">Sponsor</h2></li>
+                    <li><h2 class="menu-item">News</h2></li>
+                    <li><h2 class="cc">CC Campaign</h2></li>
+                </ul>
+            </div>
```

Comment:
- [ ] Why dont you use the `header` tag instead of the section for semantic reasons?